builtins/builtins.go

```diff
@@ -34,6 +35,15 @@ func Int2Char(args ...runtime.Object) runtime.Object {
 	return runtime.Errorf("Argument is out of range. Range is from 0 to 127. Int = %s", n.String())
 }
 
+func Str2Int(args ...runtime.Object) runtime.Object {
+	rstring := args[0].(*runtime.String)
+	i, err := strconv.Atoi(string(rstring.Value))
+	if err != nil {
+		return runtime.Errorf("invalid syntax: %s", string(rstring.Value))
+	}
+	return runtime.NewInt(int(int64(i)))
```

Comment:
`i` has already type `int` and there's no need to convert to `int64` and back:
```suggestion
	return runtime.NewInt(i)
```